packages/manager/tools/sao-ovh-manager-app/saofile.js

```diff
@@ -24,6 +24,16 @@ module.exports = {
           return `OVHcloud ${camelcase(name, { pascalCase: true })} app`;
         },
       },
+      {
+        name: 'regions',
+        message:
+          'To which region this product will be available (comma-separated values)',
```

Comment:
```suggestion
          'In which region this product will be available (comma-separated values)',
```

Reply:
Can we have a multi selector list maybe instead ?

Reply:
Multiselect list doesn't seems to be available through `sao` (even tested version `2.0.0-beta.1`)

Reply:
Seems to be available.

See:
- https://github.com/saojs/sao/blob/v1.7.0/package.json#L31
- https://github.com/SBoudrias/Inquirer.js/tree/v6.0.0#checkbox---type-checkbox